ci-operator/step-registry/ipi/aws/ipi-aws-workflow.metadata.json

```diff
@@ -2,15 +2,16 @@
 	"path": "ipi/aws/ipi-aws-workflow.yaml",
 	"owners": {
 		"approvers": [
-			"smarterclayton",
-			"wking",
-			"vrutkovs",
-			"abhinavdahiya",
-			"deads2k",
-			"crawford",
-			"ewolinetz",
-			"csrwng",
-			"staebler"
+			"kwoodson",
+			"jcpowermac",
+			"fabianofranz",
+			"rvanderp3"
+		],
+		"reviewers": [
+			"kwoodson",
+			"jcpowermac",
+			"fabianofranz",
+			"rvanderp3"
```

Comment:
Again, this is not an expected change in aws code for alibaba platform.

Reply:
Same with n+1 below.

Reply:
/hold
This PR should not be merged with current approvers and reviewers changes.

Reply:
I updated a few of these files and when `make update` runs they are reverted and updated. When I used the aliases they were removed in favor of the list of reviewers. @jstuever I'll update.